packages/@okta/vuepress-site/docs/guides/event-hook-ngrok/main/index.md

```diff
@@ -122,33 +124,33 @@ Create the Okta event hook to work with your local application, which can now be
 
 8. Click **Save & Continue**.
 
-9. With your ngrok session and local application running, complete the one-time verification Okta call at this time. You can also verify the event hook later.
+9. With your ngrok session and local application running, complete the one-time verification Okta call now. You can also verify the event hook later.
 
 ### Verify the event hook
```

Comment:
This section is a little difficult to read with the nested sections and `Or:` Could you organize this a little differently?

Reply:
It's a little excessive and, really, not necessary based on the final step in the previous procedure. Endpoint verification is discussed elsewhere for background.